package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "tc3005b-501-backend",
+    "version": "0.1.0",
+    "type": "module",
+    "description": "API y Base de Datos para la conexión y el funcionamiento del portal del sistema gestor de viajes",
```

This should be translated. And with the added information of the repository description and `README.md`.
```suggestion
    "description": "API and Database for the conection and the functioning of the trip management system portal developed in course TC3005B by group 501.",
```